src/cvodes/cvodes_io.c

```diff
@@ -737,8 +722,8 @@ int CVodeSetStopTime(void* cvode_mem, sunrealtype tstop)
   {
     if ((tstop - cv_mem->cv_tn) * cv_mem->cv_h < ZERO)
     {
-      cvProcessError(cv_mem, CV_ILL_INPUT, "CVODES", "CVodeSetStopTime",
-                     MSGCV_BAD_TSTOP, tstop, cv_mem->cv_tn);
+      cvProcessError(cv_mem, CV_ILL_INPUT, __LINE__, __func__, __FILE__,
+                     MSGCV_BAD_TSTOP);
```

Comment:
```suggestion
                     MSGCV_BAD_TSTOP, tstop, cv_mem->cv_tn);
```

Reply:
Thanks for pointing these ones out, I will fix my custom clang tool to address the extra arguments.